presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -13,49 +13,29 @@
  */
 package io.prestosql.plugin.sqlserver;
 
-import com.google.common.collect.ImmutableMap;
-import io.prestosql.testing.docker.DockerContainer;
-import io.prestosql.testing.docker.DockerContainer.HostPortProvider;
+import org.testcontainers.containers.MSSQLServerContainer;
 
 import java.io.Closeable;
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.Statement;
 
-import static java.lang.String.format;
-
 public final class TestingSqlServer
```

Comment:
Consider removing this class. I'm not sure it provides much value compared to using `MSSQLServerContainer` directly. I did that in these two and it seems to work well: #2408 #2410

Reply:
I think the small benefit is we can manage the version in a single place. We will need to fix three places in SQLServer connector when we upgrade the version.

Reply:
Agreed, I came to the same conclusion thanks to @kokosing. One approach is to make `TestingSqlServer` extend `MSSQLServerContainer`, like I did in 79b5d4f064ab934d6a53ea0972cf821a932db1b2. Then you don't need `getUsername()`, etc.